Makefile

```diff
@@ -41,22 +40,22 @@ licenses: ## Verifies dependency licenses and requires GITHUB_TOKEN to be set
 	golicense hack/license-config.hcl karpenter
 
 apply: ## Deploy the controller into your ~/.kube/config cluster
-	$(WITH_GOFLAGS) ko apply -B -f config
+	helm template karpenter ./config/ | $(WITH_GOFLAGS) ko apply -B -f -
```

Comment:
nit: can you just do `./config` or better yet even `config`

Reply:
Agreed